<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="146064" started-at="2020-03-03T19:31:30 IST" finished-at="2020-03-03T19:33:57 IST">
    <groups>
    </groups>
    <test name="Test_0" duration-ms="146064" started-at="2020-03-03T19:31:30 IST" finished-at="2020-03-03T19:33:57 IST">
      <class name="com.project1.testscripts.WebPageTestCase1">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:com.project1.testscripts.WebPageTestCase1@77847718]" name="beforeSuite" is-config="true" duration-ms="20" started-at="2020-03-03T19:31:30 IST" finished-at="2020-03-03T19:31:30 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="FAIL" signature="test1(java.lang.String)[pri:0, instance:com.project1.testscripts.WebPageTestCase1@77847718]" name="test1" duration-ms="143421" started-at="2020-03-03T19:31:31 IST" finished-at="2020-03-03T19:33:54 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Web Based application test]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect to find [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.wma.framework.common.InitTest.finallyMethod(InitTest.java:161)
at com.project1.testscripts.WebPageTestCase1.test1(WebPageTestCase1.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:483)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at org.testng.TestRunner$$Lambda$49/135640095.accept(Unknown Source)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at com.wma.framework.driver.DriverScript.startExecution(DriverScript.java:104)
at com.project1.driver.Project1Driver.main(Project1Driver.java:10)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.project1.testscripts.WebPageTestCase1@77847718]" name="afterClass" is-config="true" duration-ms="2596" started-at="2020-03-03T19:33:54 IST" finished-at="2020-03-03T19:33:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="PASS" signature="afterSuit()[pri:0, instance:com.project1.testscripts.WebPageTestCase1@77847718]" name="afterSuit" is-config="true" duration-ms="1" started-at="2020-03-03T19:33:57 IST" finished-at="2020-03-03T19:33:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuit -->
      </class> <!-- com.project1.testscripts.WebPageTestCase1 -->
    </test> <!-- Test_0 -->
  </suite> <!-- Suite -->
</testng-results>
